docs/docs/configuration/advanced.md

```diff
@@ -128,3 +128,34 @@ To do this:
 2. Rename the build to `go2rtc`.
 3. Give `go2rtc` execute permission.
 4. Restart Frigate and the custom version will be used, you can verify by checking go2rtc logs.
+
+## Validating your config.yaml file updates
+
+When frigate starts up, it checks whether your config file is valid, and if it is not, the process exits. To minimize interruptions when updating your config, you have two options -- you can either leverage the config update API endpoint, or you can validate on the command line using the frigate docker container.
```

Comment:
```suggestion
When frigate starts up, it checks whether your config file is valid, and if it is not, the process exits. To minimize interruptions when updating your config, you have three options -- you can edit the config via the WebUI which has built in validation, use the config API, or you can validate on the command line using the frigate docker container.
```

Reply:
Ha, excuse my engineer brain >.< Completely forgot about updating via the UI when writing this 😅